### PR TITLE
Allow getting data from SuppliedCache without executing the supplier

### PR DESCRIPTION
**Pull request by wyrzyk**

### CODE REVIEW

File: src/main/java/com/atlassian/db/replica/api/ThrottledCache.java [hunk 1]
```diff
@@ -23,6 +23,11 @@
         return Optional.ofNullable(value);
     }
 
+    @Override
+    public Optional<T> get() {
+        return Optional.ofNullable(value);
```
Comment (dagguh) [inline src/main/java/com/atlassian/db/replica/api/ThrottledCache.java:28]: When is it ok to skip `maybeRefresh`?
The `Supplier` isn't executed if the lock is closed.

File: CHANGELOG.md [hunk 2]
```diff
@@ -24,6 +24,18 @@ Dropping a requirement of a major version of a dependency is a new contract.
...
 [Unreleased]: https://github.com/atlassian-labs/db-replica/compare/release-1.3.0...master
 
+### Added
+- Allow getting cached value from `SuppliedCache` without executing a supplier
+
+### Changed
+
+- Make `PessimisticPropagationConsistency` class final
+- Make `ThrottledCache` class final
+- Make `NoCacheSuppliedCache` class final
```
Comment (dagguh) [inline CHANGELOG.md:34]: Breaking compatibility is ok 👌 
We just have to communicate it with a major bump. `db-replica:2.0.0` here we come!

File: src/main/java/com/atlassian/db/replica/api/ThrottledCache.java (hunk 1 above)
Comment (wyrzyk) [reply src/main/java/com/atlassian/db/replica/api/ThrottledCache.java:28]: It's always ok to read the old value from the cache.

Comment (dagguh) [reply src/main/java/com/atlassian/db/replica/api/ThrottledCache.java:28]: Hmm, that's true for `SuppliedCache`, it never guaranteed any degree of freshness.

**Review by dagguh — APPROVED** [review]

**Pull request merged**